<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SidebarWidget</class>
 <widget class="QWidget" name="SidebarWidget">
  <property name="minimumSize">
   <size>
    <width>220</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="appTitle">
     <property name="text">
      <string>Qu·∫£n l√Ω b√£i ƒë·ªó xe</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string>font-weight:bold; color:#1e3a8a; padding:12px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="nav">
     <layout class="QVBoxLayout" name="navLayout">
      <item>
       <widget class="QPushButton" name="btnDashboard">
        <property name="text"><string>B·∫£ng ƒëi·ªÅu khi·ªÉn</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSearch">
        <property name="text"><string>T√¨m xe</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMonthly">
        <property name="text"><string>V√© th√°ng</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnHistory">
        <property name="text"><string>L·ªãch s·ª≠ ra v√†o</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnParkingMap">
        <property name="text"><string>üÖøÔ∏è S∆° ƒë·ªì b√£i xe</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStatistics">
        <property name="text"><string>Th·ªëng k√™</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSettings">
        <property name="text"><string>C√†i ƒë·∫∑t</string></property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
